### DISCUSSION TITLE
Formula for Rollups? and Groupby in the view?

### DISCUSSION THREAD

**Discussion by hendrikjap** [Budibase/budibase#2763] [Ideas]
Hi Guys,

1. Is there a way to sumif / rollups data from another table? 
2. Groupby function in the view is unable to group by column created by relationship?

![image](https://user-images.githubusercontent.com/55825618/135032617-26a51ef8-fbc3-44a1-b665-026aa4b2f8c0.png)
![image](https://user-images.githubusercontent.com/55825618/135032638-680c79e7-170f-4120-aba8-76cd91fb8c0b.png)

Thanks

**Comment by mjashanks** [comment]
Not at the moment. Definitely, something we want to add (I see that I said this a few months ago too, in the other post that you commented in!).
